app/templates/admin/permissions.hbs

```diff
@@ -1 +1,11 @@
+<div>
+  {{#tabbed-navigation}}
```

Comment:
@harshitagupta30 Please use the menu similar in design to the `admin/settings` route for this sub menu.
![image](https://user-images.githubusercontent.com/17252805/27002492-894025b0-4e01-11e7-83a8-32d8de0aef68.png)
You can achieve it via the non pointing version of tabbed-navigation